wiki/Roll-Formulas.md

```diff
@@ -52,10 +48,6 @@
 
 #### Armor Class
 
-`@attributes.ac.calc` - Calculation mode that its used for determining `attributes.ac.base`
-
-`@attributes.ac.formula` - Custom formula that will be used to determine `attributes.ac.base` if `calc` is set to `custom`
```

Comment:
Why are these removed?

Reply:
They can't be used. Unsure why they were ever included; you can't construct a roll formula with the word "default", and we don't (off the top of my head) double-evaluate roll data to allow for use of `ac.formula`.